articles/azure/asz-sco

```diff
@@ -0,0 +1,262 @@
+
+The service levels are as follows:
+
+- 99.90% (a single VM deployed as a single instance deployed inside and availability zone)
+- 99.95% (a single VM with two instances deployed inside an availability zone)
+
+> [!TIP]
+> To qualify for the higher service level, customers are advised to use on platform tools like availability sets that ensures your VMs are provisioned to different hardware in the region.
+
+## Service background
+
+- VMs are fully provisioned.
+
+- The system is configured to automatically balance resources, so your VMs may be moved between physical hosts.
+
+- We actively capacity-manage the cloud platform to ensure you have access to the resources you request.
+
+- You can specify the site where you would like to have your service provisioned (from sites where UKCloud for Microsoft Azure is available). 
+
+- The UKCloud for Microsoft Azure service is updated regularly with the latest code release from Microsoft - typically, this is once a month and notified via the [UKCloud service status page](https://status.ukcloud.com/) and the [UKCloud portal](https://portal.skyscapecloud.com/login). UKCloud aim to be within two updates from Microsoft, however, we will control the release of the patches into the live service.
+
+- Through the Azure Stack portal customers can use availability sets so that VMs do not run on the same physical host.
+
+- You can make additional configurations inside a VM (such as acting as a secondary hypervisor or implementing third-party software technologies). We do not support customer implementations inside a VM.
+
+## Azure Marketplace
+
+UKCloud for Microsoft Azure includes access to a local version of the Azure Marketplace. The Marketplace contains a collection of services, applications, and resources customized for Azure Stack that customers can deploy in their environment.
+Every Marketplace item has:
+
+- An Azure Resource Manager template for resource provisioning
+- Metadata, like strings, icons, and other marketing collateral
+- Formatting information to display the item in the portal
+
+UKCloud have pre-populated the Marketplace with a variety of images and templates that have been identified as the most useful services. Customers can request additional are added, providing they are approved for use on Microsoft Azure Stack. Here is the full list of [available items](https://docs.microsoft.com/en-us/azure/azure-stack/azure-stack-marketplace-azure-items)
+
+The customer is repsonsible for ensuring that they have in place appropriate licencing and support for any item deployed from the Marketplace not licenced by UKCloud.
+
+## Operating systems
+
+**Licensing.** We can provide:
+
+- OS licensing for Microsoft
+
+- Red Hat Enterprise Linux licensing
+
+For the latest available licences, please check the *Service Definition* [xxxxx].
```

Comment:
Use the links from the Supporting documents and resources at the end of the article